build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -186,26 +186,45 @@ function build_in_tree() {
   ccache -s
 }
 
+function _check_file_not_changed_by() {
+  # _check_file_not_changed_by <cmd> <file>
+  cmd="$1"
+  file="$2"
+  file_backup="/tmp/$(basename $file)"
+  file_new="/tmp/$(basename $file).new"
+  # Save the original file.
+  cp "$file" "$file_backup"
+  # Run the command to regenerate it.
+  "$1" || return 1
+  # Save the new generated file.
+  cp "$file" "$file_new"
+  # Restore the original file. We want this function to not change the user's
+  # working tree state.
+  mv "$file_backup" "$file"
+  # We use git-diff as "just a diff program" (no SCM stuff) because it has
+  # nicer output than regular `diff`.
+  if ! git diff --quiet "$file" "$file_new"; then
```

Comment:
I think we still want @ashay feed back to redirect stderr / stdout to /dev/null.

Reply:
Not sure I understand. --quiet shouldn't output anything.

Reply:
@ashay had suggested redirecting to `&> /dev/null` in this review https://github.com/llvm/torch-mlir/pull/1318 . Maybe it is not required with `--quiet`